docs/source/api/cachegroup_fallbacks.rst

```diff
@@ -19,6 +19,10 @@
 ``cachegroup_fallbacks``
 ************************
 
+.. deprecated:: ATCv4.0
```

Comment:
I have seen in other deprecation notices that we use the API version not the ATC version. Should this match those?

Reply:
See below.